src/tool/subcommands/snapshot_cmd.rs

```diff
@@ -140,6 +157,26 @@ impl SnapshotCommands {
                 )
                 .await
             }
+            Self::Validate {
+                check_links,
+                check_network,
+                check_stateroots,
+                snapshot_files,
+            } => {
+                for file in snapshot_files {
+                    println!("Validating {}", file.display());
```

Comment:
How about
```rust
print!("Validating {} ... ", file.display());
...
println!("success");
```

Reply:
The result is displayed by ` validate_with_blockstore` like this:
  Checking IPLD integrity:       ✅ verified!
  Identifying genesis block:     ❌ No valid genesis block!  
and
  Checking IPLD integrity:       ✅ verified!
  Identifying genesis block:     ✅ found!                                                                                                                                                                                                                                                                                        
  Verifying network identity:    ✅ verified!
  Running tipset transactions:   ✅ verified!